scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -498,6 +521,13 @@ object Execution {
   private case class WriteExecution[T](head: ToWrite, tail: List[ToWrite], fn: (Config, Mode) => T) extends Execution[T] {
 
     /**
+     * override this so that we can push down the conf value to the current TypedPipes
+     */
+    override def setExtraConfig(conf: Config): Unit = {
+      (head :: tail).foreach(_.setExtraConfig(conf))
```

Comment:
Yes I'm not trying to set the Config globally, but only to the stepFlows that will be generated by this execution, see my explanation above ^^